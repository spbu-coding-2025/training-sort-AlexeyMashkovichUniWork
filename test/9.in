932842105 1039129734 -916221493 363098483 750266196 584970041 -405679219 -1025876913 634455113 -177562235 229119360 15811732 -889065447 691983198 -665192541 763674379 395121555 77911220 -950341433 364267297 382513789 -451993862 -503194829 142984401 -544331329 -345235253 -56884122 392107130 -676935016 -657436290 422717899 -817834736 -692048381 580238229 619005570 -1015524009 91466445 -860415474 -967659099 -347820267 35764114 335202084 741733289 220440491 -46556542 -997201077 -89626955 -725176812 154451966 33773435 712832308 -536776069 655521397 -864104345 679950155 -962551757 -135597775 -450675791 503297196 261209032 -34370258 -147726729 517116119 347323184 -641230325 62379864 405540998 523977944 275706214
