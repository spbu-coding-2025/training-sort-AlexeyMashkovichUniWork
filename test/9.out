-1025876913 -1015524009 -997201077 -967659099 -962551757 -950341433 -916221493 -889065447 -864104345 -860415474 -817834736 -725176812 -692048381 -676935016 -665192541 -657436290 -641230325 -544331329 -536776069 -503194829 -451993862 -450675791 -405679219 -347820267 -345235253 -177562235 -147726729 -135597775 -89626955 -56884122 -46556542 -34370258 15811732 33773435 35764114 62379864 77911220 91466445 142984401 154451966 220440491 229119360 261209032 275706214 335202084 347323184 363098483 364267297 382513789 392107130 395121555 405540998 422717899 503297196 517116119 523977944 580238229 584970041 619005570 634455113 655521397 679950155 691983198 712832308 741733289 750266196 763674379 932842105 1039129734
